distage/distage-model/src/main/scala/com/github/pshirshov/izumi/distage/model/monadic/DIEffect.scala

```diff
@@ -2,7 +2,7 @@ package com.github.pshirshov.izumi.distage.model.monadic
 
 import cats.effect.ExitCase
 import com.github.pshirshov.izumi.distage.model.monadic.FromCats._Sync
-import com.github.pshirshov.izumi.functional.bio.{BIO, BIOExit}
+import com.github.pshirshov.izumi.functional.bio.{BIO, BIOExit, F}
```

Comment:
Quite hard to notice it here, maybe move to DIEffect method. It's named as type param by purpose as I understand, but not sure if it's a good idea to mix different concepts.

Reply:
moved directly to implicit def as paramter `(implicit F: BIO[F])`